src/client/components/LeftBlueBar/LeftBlueBar.component.js

```diff
@@ -10,14 +10,14 @@ function SidebarMenu() {
         <div className="sidebar-title">GuideIT</div>
         <div className="create-account">
           <div>
-            <a href="/">Create account</a>
+            <a href="/registration">Create account</a>
```

Comment:
Would be better to use the `<Link>` component from `react-router-dom`

Reply:
ok but a question , shall I use it <Router>? because if I use <Link> alone, it will not work for storybook!

Reply:
your request is done as per you asked but I have used it inside a Router which I doubt a bit, kindly confirm that.
thank you